code/src/components/ThoughtElement.css

```diff
@@ -0,0 +1,41 @@
+.thought-element {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid black;
+    padding: 20px 20px;
+    width: 400px;
+    margin: 20px;
+    box-shadow: 4px 10px #000;
+ }
+
+ .thought-element h4 {
+    overflow-wrap: break-word
+ }
+
+ .heart {
+     background-color: #F2F0F0;
+     border-radius: 50%;
+     padding: 10px;
+     margin-right: 5px;
+ }
+
+ .thought-element button {
+     border: none;
+     background-color: white;
+ }
+
+.heart-and-time {
+     display: flex;
+     justify-content: space-between;
+     width: 100%;
+ }
+
+ .heart-and-time button {
```

Comment:
nice use of "button" as just an additional css-property. Makes for easier reading in the js as well as here.